### PR TITLE
dx-1246 Incorrect VAT info

### PR DESCRIPTION
**Pull request by arebra**
Corrected what will be displayed on the payment page.

### CODE REVIEW

File: _includes/field-description-vatamount.md [hunk 1]
```diff
@@ -3,14 +3,15 @@
...
 **`10000`** = `100.00` {{ currency }},&nbsp;
 **`5000`** = `50.00` {{ currency }}.&nbsp;
 
-If the `amount` given includes VAT, `vatAmount` may be displayed for the user in
-the payment page (redirect only). Set to `0` (zero) if there is no VAT amount&nbsp;
-charged.
+The `vatAmount` entered will not affect the `amount` shown on the payment page, 
+which only shows the total `amount`. This field is used to specify how much of
+the total `amount` the VAT will be. Set to `0` (zero) if there is no VAT
+`amount`&nbsp; charged.
```
Comment (asbjornu) [inline _includes/field-description-vatamount.md:14]: Not sure what this `&nbsp;` is doing here.

```suggestion
`amount` charged.
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: _includes/field-description-vatamount.md [hunk 2]
```diff
@@ -3,14 +3,15 @@
...
 
-If the `amount` given includes VAT, `vatAmount` may be displayed for the user in
-the payment page (redirect only). Set to `0` (zero) if there is no VAT amount&nbsp;
-charged.
+The `vatAmount` entered will not affect the `amount` shown on the payment page, 
+which only shows the total `amount`. This field is used to specify how much of
+the total `amount` the VAT will be. Set to `0` (zero) if there is no VAT
+`amount` charged.
 {%- endcapture -%}
 {{- vatamount_text | strip_newlines | replace: "&nbsp;", " " -}}
```
Comment (asbjornu) [inline _includes/field-description-vatamount.md:16]: I only now saw this replacement of `&nbsp;` with ` ` and vaguely remember why I did it. Can you confirm that everything looks fine in the portal and that every word is separated by a space as it should?

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: _includes/field-description-vatamount.md (hunk 2 above)
Comment (arebra) [reply _includes/field-description-vatamount.md:16]: I'll ask @marinafrydenb if she can do it, since she has a working docker set up.

Comment (arebra) [reply _includes/field-description-vatamount.md:16]: image

Slik ser det ut nå!

Comment (marinafrydenb) [reply _includes/field-description-vatamount.md:16]: I checked and it looks good :)

Comment (asbjornu) [reply _includes/field-description-vatamount.md:16]: I think there's missing a space between the very last `VAT` and `amount`?

**Review by asbjornu — APPROVED** [review]

**Pull request merged**